.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ refs:
   constants:
   # TODO upload to hasura docker hub:
   # - &server_builder_image hasura/graphql-engine-server-builder:2020-01-14
-  - &server_builder_image jberryman/graphql-engine-server-builder-8.10:2020-04-29
+  - &server_builder_image lexilambda/graphql-engine-server-builder:2020-08-24
```

Comment:
@lexi-lambda @jberryman BTW, both of you have push access to hasura repo in dockerhub :D

![Screen Shot 2020-08-26 at 8 42 29 PM](https://user-images.githubusercontent.com/8663570/91321968-d1fc0f80-e7dc-11ea-929e-7a5d199593b3.png)

Reply:
Yes, I intend to push the image to the hasura org before merging.